test_helpers/main.go

```diff
@@ -77,6 +85,25 @@ type TarantoolInstance struct {
 
 	// Dialer to check that connection established.
 	Dialer tarantool.Dialer
+
+	done chan error
+}
+
+// WorkStatus checks if Tarantool instance is still running.
+// Return true if it is running, false if it is not.
+// If instance was exit and error is nil - process completed success with zero status code.
+func (t *TarantoolInstance) WorkStatus() (bool, error) {
```

Comment:
This is more suitable.
```suggestion
func (t *TarantoolInstance) Running() (bool, error) {
```

Reply:
`Status` seems a bit better.